samples/Xamarin.Forms/Gallery/Gallery/WidgetPage.fs

```diff
@@ -19,8 +19,8 @@ module WidgetPageStyles =
                 let vsmGroup = VisualStateGroup(Name = "CheckedStates")
```

Comment:
I need to give a try and have VisualStateManager as a widgets / Would be awesome to have support for this :)